observability/readme.md

```diff
@@ -89,19 +89,37 @@ third-party tools like Grafana or Power BI Dashboards.
-2. Ensure health of the services are **monitored** and provide insights into system's performance and behavior.
-3. Ensure **dependent services** are monitored properly. Errors and exceptions in dependent services like Redis cache, Service bus, etc. should be logged and alerted. Also metrics related to dependent services should be captured and logged.
-4. **Faults, crashes, and failures** are logged as discrete events. This helps engineers identify problem area(s) during failures.
-5. Ensure logging configuration (eg: setting logging to "verbose") can be controlled without code changes.
-6. Ensure that **metrics** around latency and duration are collected and can be aggregated.
+1. Ensure health of the services are **monitored** and provide insights into system's performance and behavior.
+1. Ensure **dependent services** are monitored properly. Errors and exceptions in dependent services like Redis cache, Service bus, etc. should be logged and alerted. Also metrics related to dependent services should be captured and logged.
+	* Additionally, failures in **dependent services** should be propoagated up each level of the stack by the healthcheck.
+1. **Faults, crashes, and failures** are logged as discrete events. This helps engineers identify problem area(s) during failures.
+1. Ensure logging configuration (eg: setting logging to "verbose") can be controlled without code changes.
+1. Ensure that **metrics** around latency and duration are collected and can be aggregated.
 7. Start small and add where there is customer impact. [Avoiding metric fatigue](pitfalls.md#metric-fatigue) is very crucial to collecting actionable data.
-8. It is important that every data that is collected contains relevant and rich context.
-9. Personally Identifiable Information or any other customer sensitive information should never be logged. Special attention should be paid to any local privacy data regulations and collected data must adhere to those. (ex: GPDR)
+1. It is important that every data that is collected contains relevant and rich context.
+1. Personally Identifiable Information or any other customer sensitive information should never be logged. Special attention should be paid to any local privacy data regulations and collected data must adhere to those. (ex: GPDR)
 
 Read more [here](pitfalls.md) to understand what to watch out for while designing and building an observable system.
 
 ## Recipes
 
+### Service Mesh
+
+Leveraging a Service Mesh that follows the [Sidecar Pattern](https://www.oreilly.com/library/view/designing-distributed-systems/9781491983638/ch02.html#:~:text=The%20sidecar%20pattern%20is%20a,first%20is%20the%20application%20container.&text=In%20addition%20to%20the%20application,without%20the%20application%20container's%20knowledge.) quickly sets up a go to set of metrics, and traces (although traces need to be propagated from incoming requests to outgoing requests manually).
```

Comment:
https://github.com/microsoft/code-with-engineering-playbook/pull/541/commits/b1d339ea86ed4b08ad464c2455ce0e2682e5b4f1#r586146579

Reply:
Not sure what the intent of this comment is. Going to resolve, but feel free to reopen